CONTRIBUTING.md

```diff
@@ -4,7 +4,7 @@ Pull requests are welcome.
 
 ## Unit tests
 
-On a command line build the project with Maven to run the unit tests and integration tests:
+On a command line build the project with Maven to run the unit tests:
 
 ```shell
 > mvn clean test
```

Comment:
ben standardised all the repo makefiles so we can just run `make test` now rather than having to know specific maven commands